hail/src/test/scala/is/hail/expr/ir/MathFunctionsSuite.scala

```diff
@@ -38,6 +38,15 @@ class MathFunctionsSuite extends HailSuite {
     assertEvalsTo(ir, -3.0)
   }
 
+  @Test def pyUniroot() {
```

Comment:
I don't get this one.

Oh, this was used to get the right answer? Can be removed?

Reply:
Oops, I forgot to push the commit that took this out. Yeah, totally.